### PR TITLE
Remove addressable from /device api

### LINKED ISSUE #169: Changes Based on Removal of Device Addressable
This issue will contain a list of blackbox test changes that will be necessary in order to align with the effort to remove the Device/Addressable relationship. In its place, the Device will now have a `Protocols` property defined as `map[string]map[string]string`. The contract changes to support this are currently found in the following PR:
https://github.com/edgexfoundry/go-mod-core-contracts/pull/15

Once that PR is merged, then I can add the changes to edgex-go under the auspices of the following issue:
https://github.com/edgexfoundry/edgex-go/issues/1101

The following routes have been removed from the core-metadata service, so any blackbox test calling these routes should also be removed.
- /device/addressable/{addressableId}
- /device/addressablename/{addressablename}
- /deviceservice/deviceaddressables/{id}
- /deviceservice/deviceaddressablesbyname/{name}

Also, any request to a remaining route that specifies an Addressable needs to instead specify a Protocol. A very simple example that you might use in a Device JSON looks like this:
`"protocols":{"example":{"host":"localhost","port":"1234","unitID":"1"}}`

A further example that I have successfully used locally to test adding a Device with the new Protocols property based on the core-metadata-importer scripts looks like this:
```
{"origin":1471806386919,"name":"livingroomthermostat","description":"living room HVAC thermostat","adminState":"unlocked","operatingState":"enabled","protocols":{"example":{"host":"localhost","port":"1234","unitID":"1"}},"labels":["home","hvac","thermostat"],"location":"{lat:45.45,long:47.80}","service":{"name":"Gogle Home device service"},"profile":{"name":"TestProfileDeviceReport"},"autoEvents":[{"frequency":123,"onChange":true,"resource":"TestDevice"}]}
```
Note that the above JSON also contains the new `autoEvents` field per the following issue
https://github.com/edgexfoundry/edgex-go/issues/946

### PR DESCRIPTION
**Pull request by cherrycl**
Signed-off-by: Cherry Wang <cherry@iotechsys.com>

fix #169 
1. Removed device addressable related testcases
    - /device/addressable/{addressableId}
    - /device/addressablename/{addressablename}
2. Removed addressable parameter from POST /device testcases.

### CODE REVIEW

File: bin/postman-test/collections/core-metadata-importer.postman_collection.json [hunk 1]
```diff
@@ -3166,4 +3230,4 @@
 			"response": []
 		}
 	]
-}
+}
```
Comment (cloudxxx8) [inline bin/postman-test/collections/core-metadata-importer.postman_collection.json]: please add an empty line in the end of each file

File: bin/postman-test/collections/core-metadata.postman_collection.json [hunk 2]
```diff
@@ -11716,4 +11521,4 @@
 			}
 		}
 	]
-}
+}
```
Comment (cloudxxx8) [inline bin/postman-test/collections/core-metadata.postman_collection.json]: please add an empty line in the end of each file

**Review by cloudxxx8 — CHANGES_REQUESTED** [review]
please add an empty line in the end of each file

**Review by cloudxxx8 — APPROVED** [review]
LGTM

**Comment by tsconn23** [comment]
FYI I have run these test changes successfully against PR https://github.com/edgexfoundry/edgex-go/pull/1136. Will get these merged today.

**Pull request merged**